docs/src/functions/financial/fv.md

```diff
@@ -28,22 +28,22 @@ If your interest rate varies between periods, use the [FVSCHEDULE](./FVSCHEDULE)
 <!--@include: ../markdown-snippets/error-type-details.md-->
 
 ## Details
-* If *rate* = 0, FV solves the equation:
+* If *rate* = 0, FV is given by the equation:
 $$
 FV = -pv - (pmt \times nper)
 $$
 
-* If *rate* <> 0 and *type* = 0, FV solves the equation:
+* If *rate* <> 0 and *type* = 0, FV is given by the equation:
 $$ FV = -pv \times (1 + rate)^{nper} - \dfrac{pmt\times\big({(1+rate)^{nper}-1}\big)}{rate}
 $$
-* If *rate* <> 0 and *type* <> 0, FV solves the equation:
+* If *rate* <> 0 and *type* <> 0, FV is given by the equation:
```

Comment:
In some cases like the [XIRR](https://support.microsoft.com/en-us/office/xirr-function-de1242ec-6477-445b-b11b-a303ad9adc9d) function the definition of a value is through an equation:

As in rate solves the equation:

$$0 = \sum_{i=0}^N\dfrac{P_i}{\left(1+rate\right)^{\frac{d_j-d_1}{365}}}$$


PS: It's nuts that you can have LaTeX in GitHub comments!

Reply:
Looks good to me.

I will apply similar updates where required to the three pages (DATE, PV and SIN) affected by the already submitted #168 (DATE, PV and SIN).